common-app-lib/src/cmr/common_app/data/search/collection_for_gran_acls_caches.clj

```diff
@@ -81,15 +81,49 @@
                                                   :collection
                                                   elastic-item))
                           :concept-id (:_id elastic-item)))
-       query (q-mod/query {:concept-type :collection
-                           :condition q-mod/match-all
-                           :skip-acls? true
-                           :page-size :unlimited
-                           :result-format :query-specified
-                           :result-fields (cons :concept-id acl-rhh/collection-elastic-fields)
-                           :result-features {:query-specified {:result-processor result-processor}}})]
+       query (qm/query {:concept-type :collection
+                        :condition qm/match-all
+                        :skip-acls? true
+                        :page-size :unlimited
+                        :result-format :query-specified
+                        :result-fields (cons :concept-id acl-rhh/collection-elastic-fields)
+                        :result-features {:query-specified {:result-processor result-processor}}})]
   (:items (qe/execute-query context query))))
 
+(defn- fetch-collection
+ ([context collection-concept-id]
+  "Fetches one collection from elastic search by concept id. If no collection is found, will return nil."
+ (let [result-processor (fn [_ _ elastic-item]
+                         (assoc (util/delazy-all (acl-rhh/parse-elastic-item
+                                                  :collection
+                                                  elastic-item))
+                          :concept-id (:_id elastic-item)))
```

Comment:
The spacing here is still wrong.  The :concept-id is part of the assoc
`(assoc (util.delazy-all ...
              :concept-id
              (:_id ...`

Reply:
what client did this formatting? Was it by hand?

Reply:
Yes it was by hand. Although IntelliJ does have some auto formatting, which sometimes formats entire files automatically, so I turned it off. Going to back and cleaning up the formatting now

Reply:
fixed